calblueprint/vouchers4veggies
# Upgrade to Expo SDK 47, React Native 0.70.5

# ✨ New in this PR ✨
Upgraded to Expo SDK 47, which included an upgrade to React Native 0.70.5.

This is important before starting the deployment process, as Expo requires a new standalone build when bumping the Expo SDK version, so making this upgrade now will save us a re-build down the line.

I basically followed [this guide](https://blog.expo.dev/expo-sdk-47-a0f6f5c038af) to manage the upgrade. Luckily we didn't have that many parts of the app affected by any deprecations/breaking changes.

## How can the reviewer test your code? 👩‍💻
Run `npm install` and make sure everything still works as it used to, and no additional unexpected warnings are showing up. This PR should not lead to any functionality changes!

## Any bugs you encountered or still having trouble with? 🐛
n/a

## Resources 📔
https://blog.expo.dev/expo-sdk-47-a0f6f5c038af

🥦 CC: @oahnh
